chainercv/links/model/ssd/multibox_coder.py

```diff
@@ -198,7 +198,7 @@ def decode(self, mb_loc, mb_conf, nms_thresh=0.45, score_thresh=0.6):
             mb_conf (array): A float array whose shape is
                 :math:`(K, n\_fg\_class + 1)`.
             nms_thresh (float): The threshold value
-                for :meth:`chainercv.transfroms.non_maximum_suppression`.
+                for :meth:`~chainercv.utils.non_maximum_suppression`.
```

Comment:
I prefer `:func:` to `:meth:` because it is a regular function.

Reply:
OK